docs/ibm-cloud-rules.md

```diff
@@ -3770,9 +3777,9 @@ paths:
 </tr>
 <tr>
 <td valign=top><b>Description:</b></td>
-<td>Operations that return a 412 status code must support at least one of the following header parameters: <code>If-Match</code>, <code>If-None-Match</code>, <code>If-Modified-Since</code>, <code>If-Unmodified-Since</code></td>.
+<td>Operations that return a 412 status code must support at least one of the following header parameters: <code>If-Match</code>, <code>If-None-Match</code>, <code>If-Modified-Since</code>, <code>If-Unmodified-Since</code>.
 For more details, please see the API Handbook section on
-<a href="https://cloud.ibm.com/docs/api-handbook?topic=api-handbook-headers#conditional-headers">Conditional headers</a>.
+<a href="https://cloud.ibm.com/docs/api-handbook?topic=api-handbook-headers#conditional-headers">Conditional headers</a>.</td>
```

Comment:
I believe this was a typo - the formatting is odd in the doc. So I went ahead and fixed it